pkg/cmd/cli/cmd/rollback.go

```diff
@@ -221,12 +221,13 @@ func (o *RollbackOptions) Run() error {
 
 	// If this is a dry run, print and exit.
 	if o.DryRun {
-		describer := describe.NewDeploymentConfigDescriberForConfig(o.oc, o.kc, newConfig)
-		description, err := describer.Describe(newConfig.Namespace, newConfig.Name)
+		// Compare against the current deployment config
+		configName := deployutil.DeploymentConfigNameFor(target)
+		currentConfig, err := o.oc.DeploymentConfigs(o.Namespace).Get(configName)
 		if err != nil {
 			return err
 		}
-		o.out.Write([]byte(description))
+		o.out.Write([]byte(kutil.ObjectGoPrintSideBySide(currentConfig, newConfig)))
```

Comment:
Oops. Let me clarify: If we need the feature ASAP, ObjectGoPrintSideBySide is better than nothing. But if we have any time, I'd rather have something more friendly. I don't feel super strongly about it.